.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -6,9 +6,13 @@ body:
   - type: markdown
     attributes:
       value: |
-        ## FireGiant Support Customer?
+        ### FireGiant Support Customer?
 
-        If you are a [FireGiant Support Customer](https://www.firegiant.com/services/support-programs/) you can [submit bugs privately](https://support.firegiant.com/requests/new)
+        If you are a [FireGiant Support Customer](https://www.firegiant.com/services/support-programs/), you can [submit bugs and questions privately](https://support.firegiant.com/requests/new).
+
+        ### ⚠️ Everyone Else
+
+        We do NOT use these Bug Reports to ask questions or debug why your code/build doesn't work, for that, [ask a question on Discussions](https://github.com/orgs/wixtoolset/discussions/new?category=questions).        
```

Comment:
`Don't use issues to ask questions or debug why your code doesn't work; for that, ...`

Reply:
Or...

`The WiX project maintainers don't use issues for questions or for help debugging why your code doesn't work; for that, ...`

Reply:
Went with the latter.